src/BeanstalkClient.php

```diff
@@ -86,6 +87,25 @@ public function use(string $tube) {
         });
     }
 
+    public function pause(string $tube, int $delay): Promise {
+        $payload = "pause-tube $tube $delay\r\n";
+
+        return $this->send($payload, function (array $response) use ($tube): bool {
+            list($type) = $response;
+
+            switch ($type) {
+                case "PAUSED":
+                    return true;
```

Comment:
I guess it can just return `null` if the return value doesn't have any other meaning, right?

Reply:
For me, `true` here mean that operation was executed and finished successfully. Null is not really informative here, IMO

Reply:
`true` isn't either, as the successful resolution of the promise already indicates success. It's just like a `void` function in blocking PHP.

Reply:
Hmm... maybe you're right. But I'm still not sure :)

Reply:
Any opinion change?

Reply:
So you will not merge without this `true` to `null` conversion ?

Reply:
I can also change it afterwards. ;-) Could you provide a code sample that examines the return value and acts based on it?

Reply:
Any function call with remote communication that should change server state. Is not really helpful use extensions for such cases every time. 

Image that:
`function setServerAvailablity(int $code) { ... }`
will return null. Is it informative ?

Compare it with `true`